File: src/Domain/Entities/Group.cs
﻿namespace Domain.Entities;

public class Group
{
    public Guid GroupId { get; set; }
    public string Name { get; set; } = null!;
    public int Course { get; set; }
    public string Department { get; set; } = null!;
    public List<Student> Students { get; set; } = null!;
    public List<Subject> Subjects { get; set; } = null!;
}

File: src/Web/src/app/core/models/group.ts
import {Student} from "./student";

export interface GroupsResponse {
  groups: Group[]
}

export interface Group {
  groupId: string,
  name: string,
  course: number,
  department: string,
  students: Student[]
}

export interface GroupProfile {
  groupId: string,
  name: string,
  department: string,
  course: number,
}
